include/CppUTestExt/MockFailure.h

```diff
@@ -48,6 +48,7 @@ class MockFailureReporter
     virtual UtestShell* getTestToFail();
 
     virtual void crashOnFailure() {crashOnFailure_ = true; }
+    virtual void clear() {crashOnFailure_ = false; }
```

Comment:
It is very deeply encapsulated. If it wasn't, I think turning it off at the beginning of a test run would seem like a good idea.